projects/v2/src/app/static/url.ts

```diff
@@ -3,7 +3,7 @@ import { environment } from 'projects/v2/src/environments/environment';
 const PROD_URL = 'https://astb-miner-v2.herokuapp.com/v2';
```

Comment:
You should be able to remove PROD_URL and DEV_URL constants

Reply:
Done